### PR TITLE
Feature/pdf order confirmation

### PR DESCRIPTION
**Pull request by notsure**

### CODE REVIEW

File: Controller/PdfController.php [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+     * @param $id
+     * @return Response
+     */
+    public function orderConfirmationAction(Request $request, $id)
+    {
+        $locale = $this->getLocale($request);
+        $order = $this->getOrderManager()->findByIdAndLocale($id, $locale)->getEntity();
+
+        if (!$order) {
+            return new Response('', 404);
```
Comment (turbo-ele) [inline Controller/PdfController.php]: :(

Comment (turbo-ele) [inline Controller/PdfController.php]: an exception would be nice

File: Controller/PdfController.php [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+            array(
+                'baseUrl' => $this->getBaseUrl($request),
+            )
+        );
+
+        $pdf = $this->getPdfManager()->convertToPdf(
+            'SuluSalesOrderBundle:Template:order.confirmation.pdf.html.twig',
+            $data,
+            false,
+            array('footer-html' => $footer)
```
Comment (turbo-ele) [inline Controller/PdfController.php:85]: i'd prefer camel-case

Comment (notsure) [inline Controller/PdfController.php:85]: I would love to change it, but as 'footer-html' is a snappy option we can not use camel-case here

File: Order/OrderDependencyManager.php [hunk 3]
```diff
@@ -31,6 +31,15 @@ public function getName()
     }
 
     /**
+     * return url to generate Pdf's
+     * @return mixed
+     */
+    public function getPdfUrl()
```
Comment (turbo-ele) [inline Order/OrderDependencyManager.php]: didn't we remove this?

**Comment by turbo-ele** [comment]
ack

**Pull request merged**